assignments/life_expectancy/tests/test_cleaning_data_new.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from life_expectancy.data_cleaning import unpivot, convert_date_format, filter_region
+
+def test_unipivot(life_expectancy_unpivot, life_expectancy_original):
```

Comment:
Having unpivot as a private function saves you of having to test it

Reply:
ok